src/lib/cost_model/abstract_cost_model.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <memory>
+
+#include "cost.hpp"
+
+namespace opossum {
+
+class AbstractLQPNode;
+class AbstractOperator;
+class AbstractCostFeatureProxy;
+enum class OperatorType;
+
+/**
+ * Interface of an algorithm that predicts Cost for operators.
```

Comment:
Costs for Operators really. Costs for nodes is an addition that is based on the assumption into which operator a node is turned into.

As the comment continues below:
```
+ * Additionally, it can assume which operator the LQPTranslator will use for an LQPNode and thus estimate the Cost
+ * of the LQPNode as well (see estimate_lqp_node_cost()).
```

Reply:
Resolved